beagle_etl/jobs/lims_etl_jobs.py

```diff
@@ -88,19 +88,21 @@ def request_callback(request_id, job_group=None):
             recipes__contains=[recipes[0]]
         )
     except Operator.DoesNotExist:
-        logger.error("No operator defined for requestId: %s with recipe: %s" % (request_id, recipes[0]))
-        e = OperatorRequestEvent(job_group_id, "No operator defined for requestId").to_dict()
+        msg = "No operator defined for requestId %s with recipe %s" % (request_id, recipes[0])
+        logger.error(msg)
+        e = OperatorRequestEvent(job_group_id, "[CIReviewEvent] %s" % msg).to_dict()
         send_notification.delay(e)
         ci_review_e = SetCIReviewEvent(job_group_id).to_dict()
         send_notification.delay(ci_review_e)
-        raise FailedToSubmitToOperatorException("Not operator defined for recipe: %s" % recipes[0])
+        raise FailedToSubmitToOperatorException(msg)
     if not operator.active:
-        logger.info("Submitting request_id %s to %s operator" % (request_id, operator.class_name))
```

Comment:
I moved this to `msg` and passing it around. I'm assuming the intent was to have the same string get logged by logger.info and also in `FailedToSubmitToOperatorException`; originally two different strings were reported to the logger and the exception.